examples/aws-provider/composite-resources/daskhub/example-dask-hub-password.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: awsblueprints.io/v1alpha1
+kind: DaskHub
+metadata:
+  name: test-daskhub
+  namespace: default
+spec:
+  # compositeDeletePolicy: Foreground
+  compositionSelector:
+    matchLabels:
+      awsblueprints.io/configuration: password
+      awsblueprints.io/environment: test
+  resourceConfig:
+    awsProviderConfigName: aws-provider-config
+    kubernetesProviderConfigName: example-cas
+    region: us-west-2
+  parameters:
+    eksOidc: oidc.eks.us-west-2.amazonaws.com/id/123ABCDIEKEKEKEF1BK
```

Comment:
Can this value be created dynamically in the composition, or be referenced as "observe-only" resource or added to enviromentConfig?

Reply:
Yes, possible to enable that within the composition. It's meant to be usable by just specifying these required parameters.